docs/docs/tutorial.md

```diff
@@ -549,7 +549,7 @@ var CommentForm = React.createClass({
     var author = this.state.author.trim();
     var text = this.state.text.trim();
     if (!text || !author) {
-      return;
+      return false;
```

Comment:
Does this actually change anything?  I thought we started ignoring the return value of an event handler a long time ago.